packages/es-components/babel.config.js

```diff
@@ -1,11 +1,32 @@
-const wtwConfig = {
-  extractFormatMessage: false,
-  transformFormatMessage: false
-};
+module.exports = function(api) {
+  const modules = api.env('test') ? 'auto' : false;
+
+  const presets = [
+    [
+      'wtw-im',
+      {
+        extractFormatMessage: false,
+        transformFormatMessage: false,
+        env: {
+          useBuiltIns: 'usage',
+          corejs: 3,
+          modules: modules
```

Comment:
`useBuiltIns` is set to "usage" by [default in the preset](https://github.com/WTW-IM/babel-preset-wtw-im/blob/master/default-settings.js), so I'm not entirely sure if it's still needed. Same could be said with modules, since we test for `env` being set to `test` in there as well. We could save this for the next iteration, update the preset to use a cleaner modules check and update to corejs 3 before cleaning this up.